shell/platform/windows/win32_flutter_window.h

```diff
@@ -24,28 +24,29 @@
 
 namespace flutter {
 
-// A win32 flutter window.  In the future, there will likely be a
-// CoreWindow-based FlutterWindow as well.  At the point may make sense to
-// dependency inject the native window rather than inherit.
+// A win32 flutter child window used as implementatin for flutter view.  In the
+// future, there will likely be a CoreWindow-based FlutterWindow as well.  At
+// the point may make sense to dependency inject the native window rather than
+// inherit.
 class Win32FlutterWindow : public Win32Window {
  public:
-  Win32FlutterWindow();
+  // Create flutter Window for use as child window
+  Win32FlutterWindow(const int width, const int height);
+
+  // Create flutter Window for use as top-level window
```

Comment:
Why is a top-level window still a concept in the new structure?

Reply:
Fixed in next PR